validations/users.js

```diff
@@ -0,0 +1,82 @@
+const { getUsers } = require('../utils/storage');
+
+function validateUserName(userName) {
+    if (!userName) {
+        throw new Error('Error. There was no username provided.');
+    }
+}
+
+function validatePassword(password) {
+    if (!password) {
+        throw new Error('Error. There was no password provided.');
+    }
+}
+
+function findUserWithSameUserNameAndPassword(userName, password) {
```

Comment:
The `findUserWithSameUserNameAndPassword` function should not be in `validations` module

Reply:
Replaced.